.github/workflows/test.yml

```
@@ -18,7 +18,7 @@ jobs:
 
     strategy:
       matrix:
-        otp_version: [22, 23.2.5, 24.0.5]
+        otp_version: [22.3, 23.2.5, 24.3, 25]
```

```suggestion
        otp_version: [22, 23, 24, 25]
```
Use this OTP versions, will be always loaded latest minor version.